doc/development/source/orange-demo/orangedemo/OWLearningCurveA.py

```diff
@@ -280,7 +280,7 @@ def main(argv=sys.argv):
     l2.name = 'Logistic Regression'
     ow.set_learner(l2, 2)
 
-    l4 = Orange.classification.TreeLearner()
+    l4 = Orange.classification.SklTreeLearner()
```

Comment:
I checked this and it has some old imports that don't work (nothing related to this PR). I will update accordingly.